packages/neuron-wallet/src/services/transactions.ts

```diff
@@ -69,6 +69,11 @@ export default class TransactionsService {
     return transactions.find(tx => tx.hash === hash)
   }
 
+  // check whether the address has history transactions
+  public static isExist = (_address: string): boolean => {
```

Comment:
`isExist` is incorrect in English grammar.

Reply:
the meaning of “isExist address” is incorrect, use “hasTransactions” instead